.github/workflows/continuous-integration.yaml

```diff
@@ -45,24 +45,17 @@ jobs:
         ./scripts/pack.sh
 
     - name: 📤 Upload npm packages
-      uses: actions/upload-artifact@v2
+      uses: actions/upload-artifact@v3
       with:
         name: cardano-js-sdk-packages
-        path: |
-          ./cardano-js-sdk-golden-test-generator-${{ steps.package-version.outputs.current-version}}.tgz
+        path: ./cardano-sdk-*-${{ steps.package-version.outputs.current-version}}.tgz
           
-    - name: 📦 Package REPL
+    - name: 📦 Package Golden Test Generator
       working-directory: packages/golden-test-generator
       run: yarn pkg
 
-    - name: 📤 Upload Golden Test Generator exe - Linux
-      uses: actions/upload-artifact@v2
+    - name: 📤 Upload Golden Test Generator exe
+      uses: actions/upload-artifact@v3
       with:
-        name: golden-test-generator-linux
-        path: packages/golden-test-generator/build/golden-test-generator-linux
-
-    - name: 📤 Upload Golden Test Generator exe - macOS
-      uses: actions/upload-artifact@v2
-      with:
-        name: golden-test-generator-macos
-        path: packages/golden-test-generator/build/golden-test-generator-macos
\ No newline at end of file
+        name: golden-test-generator
+        path: ./packages/golden-test-generator/build/golden-test-generator-*
```

Comment:
I'll revert this to separate the downloads, given they're platform-specific

Reply:
Done in 7fc8f84b390aeaad115400cd2f38e28258eb98bf. To squash before merging to _master_